#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_rules.mk.sh.dpatch by Christian Marillat <marillat@debian.org>
##
## All lines beginning with ## DP:' are a description of the patch.
## DP:

@DPATCH@

--- librep-0.17.orig/rules.mk.sh	2004-06-17 13:36:35.000000000 +0200
+++ librep-0.17/rules.mk.sh	2004-06-17 13:33:54.000000000 +0200
@@ -12,7 +12,7 @@
 repcommonexecdir=$repcommonexecdir
 rpath_repcommonexecdir=$repcommonexecdir
 
-rep_LIBTOOL=\$(repcommonexecdir)/libtool
+rep_LIBTOOL=\$(repcommonexecdir)/libtool --tag CC
 rep_INSTALL_ALIASES=\$(repcommonexecdir)/install-aliases
 
 # use this like:
